packages/core/src/components/viewer-grid/viewer-grid.tsx

```diff
@@ -189,6 +215,9 @@ export class ViewerGrid {
     this.filterpanelFocustrap?.deactivate();
 
     this.host.removeEventListener("keydown", this.keyDownListener);
+    window
+      .matchMedia("(min-width: 768px)")
+      .removeEventListener("change", (largeScreen) => (this.tabView = !largeScreen.matches));
```

Comment:
Dit is niet hoe `removeEventListener()` werkt: `addEventListener()` en `removeEventListener()` moeten met dezelfde functie worden aangeroepen. De referentie naar de functie die bij `addEventListener()` is gegeven is bij het de-registreren nodig.

Nu geef je een nieuwe functie - die er niet geheel toevallig precies hetzelfde uit ziet -, maar het blijft een nieuwe functie met een nieuwe verwijzing.

Reply:
Ah ja, was inderdaad vergeten dit in een functie te plaatsen

Reply:
:+1: